packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -106,6 +106,11 @@ export interface IDropdownProps extends React.Props<Dropdown> {
   required?: boolean;
 
   /**
+   * Max height of dropdown options.
+   */
+  maxHeight?: number;

```

Comment:
While this works, we try to avoid having props that map directly to single CSS values. We're missing an opportunity to allow for other customizations via CSS. Could you change this prop to `optionsClassName?: string;` and have it output as the class name below?